tests/client/lwm2mclient.c

```diff
@@ -390,7 +456,7 @@ int main(int argc, char *argv[])
     }
 
     int serverId = 123;
-    objArray[3] = get_server_object(serverId, "U", 300, false);
+    objArray[3] = get_server_object(serverId, "U", livetime, false);
```

Comment:
... still a live :-) 
lifetime is much better!
I expect David will change it - right ?